### PR TITLE
Add model training script and update README with usage instructions

### LINKED ISSUE #42: Dummy/Base Model
Create a simple heuristic or "dummy" model (e.g., predicting the mean for regression, or the majority class for classification).

Record the accuracy/error of this baseline. If your model cannot beat this number, your model is not that useful.

### LINKED ISSUE #43: Training Script
You must create a new script: src/train_model.py.
Input: This script should read the clean data from data/processed.
Process: It should split the data (Train/Test), train your model (Linear Regression, Random Forest, XGBoost, etc.), and generate predictions.
Output: The script should save your trained model (using pickle or joblib) to a models/ folder (create this if it doesn't exist).

### PR DESCRIPTION
**Pull request by taylorshipley**

### CODE REVIEW

**Review by ChaseMichaelPowers — COMMENTED** [review]
Looks good

**Review by MyleeCS — DISMISSED** [review]

File: src/train_model.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            ("preprocessor", preprocessor),
+            ("regressor", LinearRegression()),
+        ]
+    )
+
+    model.fit(x_train, y_train)
+    y_pred = model.predict(x_test)
+
+    mae = mean_absolute_error(y_test, y_pred)
+    rmse = mean_squared_error(y_test, y_pred, squared=False)
```
[the remark this answers is not in the document]
Comment (MyleeCS) [reply src/train_model.py]: Problem file was deleted
